### DISCUSSION TITLE
[Feature] Ability to install Elementary OS without sight

### DISCUSSION THREAD

**Discussion by TeamLinux01** [elementary/installer#609] [Ideas]
It would be great if Elementary OS could be installed without a display, relaying on audio to guide the install. This would be wonderful for people who are partially blind or fully blind, along with useful for people with problems displaying the content of the installer, do to not having a monitor attached or very low resolution.

I would recommend that the audio feature should be turned on by default when the installer starts and make it easy to turn off (both a visual way and a keyboard combination would be nice), that way there is no stumbling to activate the feature when it is needed and can get out of the way when it is not.

Other things to consider would be detecting if audio is accessible by the installer, choosing a sane volume for the feature and making it easy to change the volume during the installation.

**Comment by danirabbit** [comment]
Hey thanks for your feedback! This seems more like an idea or a goal than an action item, so I'm going to convert it from an "Issue" to a "Discussion" to reflect that. Issues are best used for specific action items, so we can use that discussion to figure out exactly what those are and file individual issues for them
